File: ranking/src/main/java/com/example/ranking/entity/Member.java
package com.example.ranking.entity;

import jakarta.persistence.*;
import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "MEMBER")
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Setter(AccessLevel.PRIVATE)
@Getter
public class Member extends AbstractEntity {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "member_no", nullable = false)
    private Long memberNo;

    @Column(name = "member_name", nullable = false)
    private String memberName;

    @Column(name = "score", nullable = false)
    private Long score;

    public static Member of(String memberName, Long score) {
        Member member = new Member();
        member.setMemberName(memberName);
        member.setScore(score);
        return member;
    }
}

File: mysql/src/main/kotlin/com/example/mysql/repository/Member.kt
package com.example.mysql.repository

import jakarta.persistence.*

@Entity
@Table(name = "MEMBER")
class Member(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "member_no", nullable = false)
    var memberNo: Long = 0,

    @Column(name = "name", nullable = false)
    var name: String
) {
}